webpack/ForemanTasks/Components/common/ActionButtons/ActionButton.js

```diff
@@ -1,27 +1,38 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { CancelButton } from './CancelButton';
-import { ResumeButton } from './ResumeButton';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { ActionButtons } from 'foremanReact/components/common/ActionButtons/ActionButtons';
 
 export const ActionButton = ({ id, name, availableActions, taskActions }) => {
   const isResume = availableActions.resumable;
+  const isCancel = availableActions.cancellable;
+  const buttons = [];
+  let title;
+  if (!isResume && !isCancel) {
+    title = __('Task cannot be canceled');
```

Comment:
looks like `title` is being set only here,
so I assume we could use a `const` instead

Reply:
Can't because then title will be considered undefined by eslint (as its only defined inside the if block)

Reply:
you can do something like:
```js
const title = !isResume && !isCancel ? __('Task cannot be canceled') : null
```

Reply:
Changed, thanks